mk/config.mk

```diff
@@ -1132,3 +1132,10 @@ $(eval $(call cfg-depends-all,CFG_CRYPTO_HW_PBKDF2,CFG_CRYPTO_PBKDF2))
 CFG_HALT_CORES_ON_PANIC ?= n
 CFG_HALT_CORES_ON_PANIC_SGI ?= 15
 $(eval $(call cfg-depends-all,CFG_HALT_CORES_ON_PANIC,CFG_GIC))
+
+# CFG_DRIVERS_REMOTEPROC, when enabled, embeds support for remote processor
+# management including generic DT bindings for the configuration.
+CFG_DRIVERS_REMOTEPROC ?= n
+
+# CFG_RPROC_PTA, when enabled, embeds remote processor management PTA service.
+CFG_RPROC_PTA ?= n
```

Comment:
Would you mind `s/CFG_RPROC_PTA/CFG_REMOTEPROC_PTA/` for consistency with `CFG_DRIVERS_REMOTEPROC` (and better "grepability", too)?

Reply:
yes much better  I also move the declaration in the patch that introduce the remoteproc PTA